lib/driver.rb

```diff
@@ -43,5 +44,67 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def get_total_revenue
+      total_rev = 0.0
+      @trips.each do |trip|
```

Comment:
It's Enumerable time! https://ruby-doc.org/core-2.1.0/Enumerable.html#method-i-reduce